client/state/data-layer/wpcom/jetpack-onboarding/index.js

```diff
@@ -59,6 +117,13 @@ export const announceSaveFailure = ( { dispatch } ) =>
 	dispatch( errorNotice( translate( 'An unexpected error occurred. Please try again later.' ) ) );
 
 export default {
+	[ JETPACK_ONBOARDING_SETTINGS_REQUEST ]: [
+		dispatchRequest(
```

Comment:
definitely checkout `dispatchRequestEx()` which @jsnajdr introduced because it can remove needless code here, like the actual `dispatch`es

Reply:
For better or worse, we're currently requiring `getState()` in our `http()` fetch action, which `dispatchRequestEx()` doesn't seem to support AFAICS, so it looks like we're stuck with `dispatchRequest()` for now :slightly_frowning_face: